chains/scripts/add_users.js

```diff
@@ -0,0 +1,50 @@
+async function loadem(starting_page = 0) {
```

Comment:
Hm what is this file actually?

Reply:
I think Coburn was using this for... load testing?